repository.py

```diff
@@ -36,18 +43,18 @@ def get_current_player():
   #Gets the player whose turn it is
   state = get_game_state()
   return state['currentPlayer']
-  
+
 def advance_round(new_current_player):
   #set a new current player and increment the turn counter
   db = get_connection()
   gameDB = db.game
-  gameDB.update_one({"_id" : "gameState"}, 
+  gameDB.update_one({"_id" : "gameState"},
                     {"$set" : {"currentPlayer" : new_current_player}, "$inc" : {"currentRound" : 1}})
 
 def init_db(first_player):
   db = get_connection()
   gameDB = db.game
-  gameDB.insert_one({"_id" : "gameState", 
+  gameDB.insert_one({"_id" : "gameState",
```

Comment:
Yeah this guy here is what breaks. Mongo doesnt like trying to insert a second document with the same key.

We could change it to an upsert (like what we do with adding players) and that would avoid the duplicate key errors. Or we can check the database to see if its been updated instead of relying on flags in config.json